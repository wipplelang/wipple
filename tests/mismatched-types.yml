code: |
    42 :: Text

output: |

diagnostics: |
    error: mismatched types
      ┌─ test:1:1
      │
    1 │ 42 :: Text
      │ ^^ expected `Text`, but found `Number`
      │
      = for more information, see https://wipple.dev/playground/?lesson=debugging/mismatched-types
