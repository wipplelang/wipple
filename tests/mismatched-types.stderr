error: mismatched types
  ┌─ test:1:1
  │
1 │ 42 :: Text
  │ ^^    ---- expected because the type was annotated here
  │ │      
  │ expected `Text`, but found `Number`
  │
  = for more information, see https://wipple.dev/playground/?lesson=errors/mismatched-types

